feeds/npm/npm.go

```diff
@@ -75,19 +50,49 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
```

Comment:
This is slightly misleading now since we're only extracting the PKG name from title. Should we just return a collection of strings?

Reply:
My personal preference would be that this func still generates the package structs, and then the version info is added in the seperate func. Happy to change it though if needs be.

Reply:
Maybe some documentation could be added to the func to make it clear that the package is partially populated and is not sufficient without additional processing.

Reply:
Sure, done!